web-dev/Frontend/Libraries & Frameworks/Learn-React.md

```diff
@@ -41,6 +41,9 @@
   * [React without ES6](#react-without-es6)
   * [Reconcilation - Diff Algorithm](#reconcilation---diff-algorithm)
 
+* [Complete Projects](#complete-projects-that-you-can-have-a-look)
```

Comment:
I dont think it is complete so lets remove the title `complete projects` and let it be just `open source projects you can contribute to`